src/main/java/io/orten/nano/business/UserAPI.java

```diff
@@ -0,0 +1,74 @@
+package io.orten.nano.business;
+
+import io.orten.nano.impl.UserService;
+
+import io.orten.nano.model.User;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+@Path("/")
+public class UserAPI {
+
+    public static List<User> u_list = new ArrayList<User>();
+
+    @GET
+    @Path("/get/user/{userID}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getUser(@PathParam("userID") long userID) {
+        try {
+            User user = UserService.getUser(userID);
+            return Response.status(200).entity(user).build();

```

Comment:
Switch from using static integers to HttpServletResponse.SC_OK

Reply:
Replaced 200 by HttpServletResponse.SC_OK 
and 
Replaced 500 by HttpServletResponse.SC_INTERNAL_SERVER_ERROR